plugins/community/nanopb/v0.4.9/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# syntax=docker/dockerfile:1.10
+FROM python:3.11.10-alpine3.20 AS build
```

Comment:
more drive-by: when do these python versions get bumped? 3.12 has been out since October 2023; https://github.com/docker-library/python/blob/9cd324371ef929bfc5036a00d7705a213d8b5020/3.12/alpine3.20/Dockerfile

Reply:
I don't think we have a hard rule, I checked upstream and noticed they were still using `3.11`.

https://github.com/nanopb/nanopb/blob/nanopb-0.4.9/MODULE.bazel#L17

And we have this in our .github/docker/Dockerfile.python for python

https://github.com/bufbuild/plugins/blob/5bdb46fdb62e7bd0b954250175491c893e008189/.github/docker/Dockerfile.python#L1

So I think it is fine to leave

Reply:
I've left it on 3.11 as that's the version packaged in the distroless images:

```
$ docker run --pull=always -it --rm gcr.io/distroless/python3-debian12:latest --version           
latest: Pulling from distroless/python3-debian12
Digest: sha256:55283e240bb5bd506e711a395092b61acf1fb14e63977ee24336285dd9e68fb8
Status: Image is up to date for gcr.io/distroless/python3-debian12:latest
Python 3.11.2
```